ssmconfig.go

```diff
@@ -48,59 +54,56 @@ type Provider struct {
 // The behavior of using the `default` and `required` tags on the same struct field is
 // currently undefined.
 func (p *Provider) Process(configPath string, c interface{}) error {
-
```

Comment:
Please remove all of the unnecessary white space. Again, this pollutes the diff.